iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/listener/api/event/IcebergRequestContext.java

```diff
@@ -84,6 +98,15 @@ public Map<String, String> httpHeaders() {
     return httpHeaders;
   }
 
+  /**
+   * Checks if the request is for credential vending.
+   *
+   * @return true if the request is for credential vending, false otherwise.
+   */
+  public boolean isRequestCredentialVending() {
```

Comment:
Can we rename to `requestCredentialVending`?

Reply:
done